README.md

```diff
@@ -458,7 +458,7 @@ cdnvm(){
         nvm_version=$(<"$nvm_path"/.nvmrc)
 
         # Add the `v` suffix if it does not exists in the .nvmrc file
-        if [[ $nvm_version != v* ]]; then
+        if [[ $nvm_version != v* && $nvm_version != lts/* && $nvm_version != iojs* && $nvm_version != node && $nvm_version != system && $nvm_version != stable && $nvm_version != default ]]; then
```

Comment:
since nvm is available, how about this:
```suggestion
        nvm_version="$(nvm_format_version "${nvm_version})"
```
instead of the whole if block

Reply:
`nvm_format_version` doesn't work for non-numeric version names.

```
$ nvm_format_version node
node.0.0
$ nvm_format_version lts/*
lts/*.0.0
```

Reply:
It would, however, make sense to replace `nvm_version="v""$nvm_version"` with `nvm_version="$(nvm_format_version "${nvm_version})"`, which is what I think you mean. So I will make those changes now :smile:

Reply:
Actually, if we do that it will introduce inconsistent behaviour. Right now, if I run `nvm install 6`, it will install the _latest_ version of '6'.

```
$ nvm install 6
Downloading and installing node v6.17.0...
Downloading https://nodejs.org/dist/v6.17.0/node-v6.17.0-linux-x64.tar.xz...
################################################################################################################################################################################################################################ 100.0%
Computing checksum with sha256sum
Checksums matched!
Now using node v6.17.0 (npm v3.10.10)
```

If we use `nvm_format_version` here, `6` would mean the _first_ version of '6'.

```
$ nvm_format_version 6
v6.0.0
```

So I think we should not use `nvm_format_version` here. @ljharb WDYT?

Reply:
ah, true. hmm. why is the `v` needed, then?

Reply:
Because later on there's a comparison made between `$(nvm current) != "$nvm_version"`, and `nvm current` outputs a `v`. There's a better way to do this, I will make another commit.

Reply:
Previously, if `$nvm_version` was, say, `8`, and `nvm current` outputs `v8.15.1`, the comparison would return `false` although it should be `true`. With commit 20bbb090746a71544be74da9c18ef7e20c05b184, this should be fixed.